atlasdb-console/src/main/groovy/com/palantir/atlasdb/console/module/AtlasCoreModule.groovy

```diff
@@ -217,7 +218,7 @@ class AtlasCoreModule implements AtlasConsoleModule {
     private setupConnection(AtlasDbConfig config) {
         TransactionManager tm = TransactionManagers.builder()
                 .config(config)
-                .userAgent("atlasdb console")
+                .structuredUserAgent(UserAgent.of(UserAgent.Agent.of("atlasdb console", "0.0.0")))
```

Comment:
This is not a valid user agent, space not allowed

Reply:
Someday, we'll have better Atlas Console tests